docs/dev/plugins/target-artifact-conventions/target-artifact-conventions.md

```diff
@@ -17,8 +17,41 @@ The purpose of this document is to specify conventions for the target artifact t
 ## Must have
 - `Readme.html` - a Readme file in the root folder of the target artifact is mandatory. It holds the basic information of the target artifact, also if the plugin needs a short usage guide include it in this file.
 - `output` - the `output` - folder holds everything generated during the transformation.
-  - `output/scripts` - `scripts` is a subfolder of `output` it holds executable scripts generated during the transformation, that are needed to deploy the artifact. Scripts are all lowercase and to seperate logical words use `-`.
+  - `output/scripts` - `scripts` is a subfolder of `output` it holds executable scripts generated during the transformation, that are needed to deploy the artifact. In the [scripts](#scripts) section you can find more about script conventions.
 
 ## Recommended
 
 - `docs` - if the transformation artifact needs a more advanced documentation put it into the `docs` folder. And link to it in the `Readme.html`. To allow the user to browse the docs they are written in HTML.
+
+
+## Scripts
+
+As mentioned before our target artifact folder has a `scripts` folder.
+To prevent random names scripts also have to follow conventions.
+
+### Script conventions
+- Scripts are all lowercase and to seperate logical words use `-`.
+- Scripts need to have the following format:
+  > `<action>-<short-title>.sh`
```

Comment:
`<short-title>` should probably have an additional explanation: e.g. short name of the object on which the action is performed on, or something similar